Assets/Scripts/Handler/Button/PauseHandler.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public class PauseHandler : MonoBehaviour
+{
+    [SerializeField] private GameObject pauseMenuUI;
+
+    public void PauseButton_OnClick()
```

Comment:
I think that `Button_OnClick` should be omitted because the name of this class implies that it's not only for a button. (Other handler classes are similar.)

Reply:
Agreed that the naming should be adjusted, but for clarity, each of these handlers are handling an event based upon a button click so they should still be handlers right?

Reply:
I think it's fine for these scripts to still be named handlers.